tests/test_utilities.py

```diff
@@ -55,7 +55,7 @@ def test_empty_df():
     [
         (b"abcde", "utf-8"),  # straight up ascii is a subset of unicode
         (b"Eyjafjallaj\xc3\xb6kull", "utf-8"),  # actual unicode
-        (b"\xC4pple", "WINDOWS-1252"),  # non-unicode, ISO characterset
+        (b"\xC4pple", "ISO-8859-1"),  # non-unicode, ISO characterset
```

Comment:
chardet and cchardet no longer detect the same encoding in this case. at least on py 3.10

Reply:
ah looks like this was changed in a "recent" pr

https://github.com/remix/partridge/pull/61/files